miso-web/src/main/webapp/scripts/hot_library.js

```diff
@@ -61,6 +61,20 @@ HotTarget.library = (function() {
             'source': indices
           });
           setData(data);
+        } else if (flatProperty === 'index' + n + 'Label'){
+          var pt = getPlatformType(flat.platformType);
+          indexFamily = Utils.array.findFirstOrNull(function(family) {
+            return family.name == flat.indexFamilyName && family.platformType == pt;
+          }, Constants.indexFamilies);
+          var indices = (Utils.array.maybeGetProperty(indexFamily, 'indices') || []).filter(function(index) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "indices" as this name is already used in declaration at line 48. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS2814)